sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/atoms/ButtonScreen.kt

```diff
@@ -46,10 +49,12 @@ fun ButtonScreen() {
     }
 }
 
-@Preview(showBackground = true)
+@HmrcAllDevicePreview()
 @Composable
 fun ButtonScreenPreview() {
     HmrcTheme {
-        ButtonScreen()
+        HmrcSurface {
+            ButtonScreen()
+        }
```

Comment:
I think we are using this `HmrcTheme { HmrcSurface { Content() } }` for all previews so maybe we can add a help function to avoid repetition in the `ComposableTemplate` file.